src/components/InspectorPage/InspectorPage.tsx

```diff
@@ -45,26 +45,31 @@ export default class InspectorPage extends React.PureComponent<Props, State> {
       binIndexJsUrl = `${htmlUrl}/bin/index.js`
     }
 
+    let queryParams = `?dataLayerRpcParentUrl=${window.location.origin}`
+
     // use the local bin/index.js being watched an served on your machine
     if (process.env.REACT_APP_BIN_INDEX_JS_DEV_PORT && process.env.REACT_APP_BIN_INDEX_JS_DEV_PATH) {
       const b64 = btoa(process.env.REACT_APP_BIN_INDEX_JS_DEV_PATH)
       binIndexJsUrl = `http://localhost:${process.env.REACT_APP_BIN_INDEX_JS_DEV_PORT}/content/contents/b64-${b64}`
     }
 
+    queryParams = queryParams.concat(`&binIndexJsUrl=${binIndexJsUrl}`)
+
+    if (process.env.REACT_APP_CATALOG_SERVER_PORT) {
```

Comment:
could we make this to te `REACT_APP_CATALOG_URL`? This way we could use `REACT_APP_CATALOG_SERVER_PORT=https://builder-items.decentraland.zone` for example, to use .zone assets but not having to launch the local asset-packs services